client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Site } from '../types';
+import SitePreviewPaneContent from './site-preview-pane-content';
+import SitePreviewPaneFooter from './site-preview-pane-footer';
+import SitePreviewPaneHeader from './site-preview-pane-header';
+import SitePreviewPaneTabs from './site-preview-pane-tabs';
+
+import './style.scss';
+
+interface Props {
+	selectedSite?: Site | undefined;
```

Comment:
The site should be a mandatory prop. So `?` and `undefined` wouldn't be necessary.

Reply:
[6f8d501](https://github.com/Automattic/wp-calypso/pull/87760/commits/6f8d501ad23e3b12787727be644dd3875568fbed)